### ISSUE TITLE
generate random (uuid ?) filename on unpacking

### ISSUE THREAD

**Issue by chipitsine** [cuckoobox/cuckoo#627]
have a look:

https://malwr.com/analysis/MDlhZTQzMTg4Y2Q3NDNhNWE3OWMxODcwYjMxMDFkZjY/

filename "InvoiceFaker::Number.number(5)info_247694448826.exe" is valid for ext4, but is not valid for ntfs.

maybe some ntfs filename validation should be added or uuid filename instead of original ?

**Comment by jbremer** [comment]
Indeed, colons are not allowed filenames on ntfs. But like, how is one supposed to run such a file on Windows anyway? If an attacker used that filename, it wouldn't run either.

**Comment by chipitsine** [comment]
windows is not capable to store such files on ntfs. however it allows such files inside so called "zip" folders.

**Comment by botherder** [comment]
Maybe that was a name that was given to the researcher that uploaded the file?
Either way, it's true, we should perhaps sanitize files in order to make them compliant to NTFS rules.
